satellite/console/service.go

```diff
@@ -894,11 +897,19 @@ func (s *Service) CreateProject(ctx context.Context, projectInfo ProjectInfo) (p
 		s.log.Debug(fmt.Sprintf("could not add promotional coupon for user %s", auth.User.ID.String()), zap.Error(Error.Wrap(err)))
 		return p, nil
 	}
-	if len(cards) == 0 {
+
+	balance, err := s.accounts.Balance(ctx, auth.User.ID)
+	if err != nil {
+		s.log.Debug(fmt.Sprintf("could not add promotional coupon for user %s", auth.User.ID.String()), zap.Error(Error.Wrap(err)))
+		return p, nil
+	}
+
+	if len(cards) == 0 && balance < s.minCoinPayment {
```

Comment:
this might work strange when we will enable multiple project per user

Reply:
We will rework this entire code after prod launch anyway.